data/json/body_parts.json

```diff
@@ -600,6 +600,103 @@
     "bmi_encumbrance_threshold": 20,
     "bmi_encumbrance_scalar": 0.15
   },
+  {
+    "id": "gastropod_foot",
+    "type": "body_part",
+    "name": "gastropod foot",
+    "accusative": { "ctxt": "bodypart_accusative", "str": "gastropod foot" },
+    "heading": "gastropod foot",
+    "heading_multiple": "gastropod feet",
+    "hp_bar_ui_text": "G. FOOT",
+    "encumbrance_text": "Running and swimming are slowed.",
+    "encumbrance_threshold": 20,
+    "main_part": "gastropod_foot",
+    "connected_to": "torso_waist",
+    "opposite_part": "gastropod_foot",
+    "limb_types": [ [ "leg_l", 0.8 ], [ "leg_r", 0.8 ], [ "foot_l", 0.5 ], [ "foot_r", 0.5 ], [ "tail", 0.2 ] ],
+    "limb_scores": [ [ "reaction", 0.3 ] ],
```

Comment:
You want move_speed (little), crawl(more), footing(probably a lot) instead of reaction.